<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloadResetWidget</class>
 <widget class="QWidget" name="MusicDownloadResetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>温馨提示</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>270</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>温馨提示</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>270</width>
     <height>115</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:rgba(255,255,255,200);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QPushButton" name="openDetailButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>查看详情</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>已下载过该歌曲哦</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>重新下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDirButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>打开目录</string>
    </property>
   </widget>
   <zorder>downloadName</zorder>
   <zorder>openDetailButton</zorder>
   <zorder>downloadButton</zorder>
   <zorder>openDirButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
